api/api/models/media.py

```diff
@@ -6,7 +6,7 @@
 from django.db import models
 from django.utils.html import format_html
 
-from elasticsearch import Elasticsearch, TransportError
+from elasticsearch import BadRequestError, Elasticsearch, NotFoundError
```

Comment:
TransportError changed and it's much easier/clearer to catch the specific errors we care about.

Realising now that we can change this to only import BadRequestError and ignore NotFoundError completely, just letting it raise the way we want it to, now that we're not catching a unified error type.

Reply:
Cleaned up in https://github.com/WordPress/openverse/pull/3029/commits/fe96cb145ddd7ea8548805411b208dfd8ab47c7b